docs/tutorials/testing_with_control_center.rst

```diff
@@ -145,16 +145,26 @@ below shows an example of what the Control Center will look like when connecting
 
 Updating Firmware
 =================
-If you do need to update the motor firmware, you can do it through the Control Center. First download the firmware binary you want to update to, and then open the Firmware tab. 
-Click on "Select Firmware Binary", and select the binary you downloaded. Make sure you are connected to the motor, and then hit "Flash". The motor should disconnect, and a progress
-bar at the bottom of the Control Center should start filling. When the progress bar is full, the flash is complete. The motor should restart and play its 5 beep startup song. An 
-example of the Control Center after a successful flash is shown below.
+If you do need to update the motor firmware, you can do it through the Control Center. First, download the most recent firmware from vertiq.co, either .zip or .bin, then open the Firmware tab. 
```

Comment:
You may want to specify the version that this information becomes true for, something like "The following instructions are intended for version 1.3.0 and later. Earlier versions have a different interface". To go one step further, you may even want to keep at least the text of the instructions for the older version for a little while. Like "For versions less than 1.3.0 follow these instructions, for 1.3.0 and later, follow these". Customers are not always eager to actually update when we give them the choice, so there may be stragglers on older versions for a bit